src/diffusers/schedulers/scheduling_ddpm_flax.py

```diff
@@ -222,9 +222,13 @@ def step(
         t = timestep
 
         if key is None:
-            key = jax.random.PRNGKey(0)
+            key = jax.random.key(0)
 
-        if model_output.shape[1] == sample.shape[1] * 2 and self.config.variance_type in ["learned", "learned_range"]:
+        if (
+            len(model_output.shape) > 1
```

Comment:
why are we adding this `len() > 1` check here?

Reply:
Because input could have shape like `x.shape == (10,)`. Accessing index `1` in the next line is an error in that case.